app/views/card_order_exports/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= render("waste_carriers_engine/shared/back", back_path: "/bo") %>
+
+    <h1 class="govuk-heading-l">
+      <%= t(".heading") %>
+    </h1>
+    <% if @exports.last.present? %>
+      <h2 class="govuk-heading-m">The latest file was created at <%= @exports.last.exported_at %></h2>
+    <% end %>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+
+    <table class="govuk-table" aria-label="<%= t(".heading") %> >
```

Comment:
Looks like Sonarcloud would like a `caption` or `aria-describedby` for this. Can we check with Will for text?

Reply:
I added a caption, will check with Will on the wording.